cmd/river/go.mod

```diff
@@ -1,16 +1,17 @@
 module github.com/riverqueue/river/cmd/river
 
 go 1.22.0
```

Comment:
should we think about bumping this next release, or not until really needed?

Reply:
Ah yeah, good question. I try not to be too aggressive about it and just bump when we want it or someone asks for it. That said, Go 1.24's been out a while now, so I think it could be plausible be done at any time.